client/src/views/CellView.tsx

```diff
@@ -101,20 +123,61 @@ const CellView = () => {
     });
   }, [Alert, Confirm, brute, t]);
 
+  useEffect(
+    () => {
+      if (user && brute && user.id === brute.userId) {
```

Comment:
It's better to check the opposite and return, to avoid wrapping the whole code in `{}`